pkg/scaffold/v2/certmanager/kustomize.go

```diff
@@ -0,0 +1,52 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package certmanager
+
+import (
+	"path/filepath"
+
+	"sigs.k8s.io/kubebuilder/pkg/scaffold/input"
+)
+
+// Kustomization scaffolds the kustomizaiton in the certmanager folder
+type Kustomization struct {
```

Comment:
We can keep certmanager scaffolders in the same file. Its easier to maintain.

Reply:
Marking this comment as resolved per https://github.com/kubernetes-sigs/kubebuilder/pull/644#issuecomment-488759892.